.travis.yml

```diff
@@ -103,6 +102,7 @@ jobs:
             secure: bk3oSSuNgpAFlEY9lyvPL/4eJoO0eUVz8Yh4gciOK+YFW8vCpH6c33psKd50QAxZi5sH2DhGN/ZfMwpZsX6x1jFDP7A4eRNh5hjAKaJmgLHigEzvp8rLgzQTJKafT/ay3HQclQyTTVT6GqO2b39pXHVttfY7XzmfQkwR9j9r+hfA/ktyl/1Mka32lPJokakU20sTvARr/pdeq0l5m2EJjJM5YIUuzjbd9D3t52va87eDzfOw64SPy1BdM0ZYaM2iZloFQg+gyGYLP0DX2qN99yKT9WyfVbaxrUcIY1kgMoWC9W7p/hyfU0qtHnvEX8W5sE7l4nfIfWOY3oclYUwV8wIPDqaB7tTldBgNWb5c1pGa3VnfFhzaToD7V7UYj7M9IDCoeBJK07kaHN2a3iYgETv/IkjzybXq42hSbomkv+iSD7iCvM5ewFEVJ6n0WQp9Q/uNrbpDwdMQ5ONIILP2KYmtNCFRIHkIBwYR2ku9UMrRnsyjOXCx49EkibwXkvrv4TAH9L8f7K6WeTxIpwGkQ+NrzX3FzWb9beYRa96yKkbfW3Hm196SyPSeE4qqd3ilaKixvtE1l8XqJt0QmdrGiyDKts17p5pVNulqyIuLrcOkXGk7JmzpOXkXpT9P88NacugBH/4DuM9Z/p+nm+pUtxjPDAxtAL2rgoufBjRlWNo=
```

Comment:
This should need updating also, travis encryption is linked to the repo

Reply:
Then you'd need to do it. But, I'm not sure about that because it wouldn't work back in YakDriver so makes me wonder if they somehow update it to work when transferred? 🤔

Reply:
No, if you copied the value from elsewhere rather than generating it yourself, then it was never valid for this project. I'll try to remember how I did this before...

Reply:
Just before the transfer, it was working fine.

Reply:
It's here: https://docs.travis-ci.com/user/encryption-keys/

Reply:
It's an encrypted github API token. I can generate one for a YakDriver token.